packages/calcite-components/src/components/link/link.e2e.ts

```diff
@@ -320,4 +322,47 @@ describe("calcite-link", () => {
       );
     });
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-link-text-color": [
+          {
+            shadowSelector: "span",
+            targetProp: "color",
+          },
+        ],
+        "--calcite-link-background-color": [
+          {
+            shadowSelector: "span",
+            targetProp: "backgroundColor",
+          },
+        ],
```

Comment:
nitpick. `--calcite-link-text-color` & `--calcite-link-background-color` don't have to be arrays